src/app.ts

```diff
@@ -39,7 +40,7 @@ export class App {
   intervalId: any;
 
   errorHandler = (ex: unknown): boolean => {
-    this.eventAggregator.publish("handleException", new EventConfigException("Sorry, an unexpected error occurred", ex));
+    this.eventAggregator.publish(EventType.HandleException, new EventConfigException("Sorry, an unexpected error occurred", ex));
```

Comment:
our practice is that changes unrelated to what is being addressed by a PR should be done in a separate PR.  Can you please create a ticket for the `EventType` enum and put it in the Triage column (the default "pipeline") of the kanban, and back the change out of this PR?